ansible_base/activitystream/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+
+from ansible_base.activitystream.models import Entry
+
+admin.site.register(Entry)
```

Comment:
I have a ReadOnlyAdmin here, might be good to use here.

https://github.com/ansible/django-ansible-base/blob/19a3e55bc70a060f49972ec45aa4024ea22cca3c/ansible_base/rbac/admin.py#L6

Reply:
Yep. I'll pull this out to a util class and use it in both places.